### ISSUE TITLE
Feather Huzzah filesystem problems after 2.0.0-beta1

### ISSUE THREAD

**Issue by dhalbert** [adafruit/circuitpython#207]
The 2.0.0-beta1 release filesystem operations work:
```
Adafruit CircuitPython 2.0.0-beta.1 on 2017-08-27; ESP module with ESP8266
>>> import os
>>> os.listdir()
['boot.py', 'test']
>>> os.mkdir('test1')
>>> os.listdir()
['boot.py', 'test', 'test1']
```
but the current tip of master does not:
```
Adafruit CircuitPython 2.0.0-beta.1-15-g4e63d55d8 on 2017-08-27; ESP module with ESP8266
>>> import os
>>> os.listdir()
[]
>>> os.mkdir("test2")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
OSError: [Errno 17] EEXIST
```
(This is before the v1.9.2 merge.)

**Comment by dhalbert** [comment]
@tannewt I am starting to look at this but if you have a hunch let me know.

**Comment by dhalbert** [comment]
git bisect shows it's commit f6a702538 (addition of ".frozen" stuff).

**Comment by dhalbert** [comment]
Fixed by #222
